projects/plutus/inc/ltc_afe.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include <stdbool.h>
+#include <stdint.h>
+#include "gpio.h"
+#include "spi.h"
+#include "status.h"
+
+// number of devices in daisy chain (including master)
+#define LTC_AFE_DEVICES_IN_CHAIN                1
```

Comment:
This is used to allocate memory for a buffer on the stack when receiving SPI data. Defining a "maximum devices supported" seems a little weird to me, since it's an arbitrary number that doesn't reflect physical limitations imposed by the hardware. If someone feels strongly about this, I can arbitrarily pick a number..

I'm leaning more towards a config header&mdash;do we have a specific place for those?